Contributing.md

```diff
@@ -87,7 +87,11 @@ create a pull request to `shuosc/master`, and check if your code can pass all th
 
 We'll check the pull request now and then, you may also request a review manually if you like.
     
+## Learning Material
+Below is something useful to know how to contribute this project, hope it would help you on your walkthrough
+#### Documentation
+* [RISCV Spec. Volumn I: User-Level ISA RV32I v2.0](https://content.riscv.org/wp-content/uploads/2019/06/riscv-spec.pdf)
```

Comment:
Please update the Spec to [the latest version](https://github.com/riscv/riscv-isa-manual/releases/download/Ratified-IMAFDQC/riscv-spec-20191213.pdf)